src/persist/src/indexed/encoding.rs

```diff
@@ -19,15 +19,17 @@ use std::fmt;
 use std::io::Cursor;
 use std::ops::Range;
 
+use bytes::BufMut;
 use differential_dataflow::trace::Description;
-use ore::cast::CastFrom;
-use persist_types::{Codec, ExtendWriteAdapter};
-use protobuf::{Message, MessageField};
 use semver::Version;
 use serde::{Deserialize, Serialize};
 use timely::progress::{Antichain, Timestamp};
 use timely::PartialOrder;
 
+use ore::cast::CastFrom;
```

Comment:
did you intentionally make this a new block of imports?

Reply:
Yeah elsewhere we try to separate into three blocks: stdlib, third-party, first-party. It'd be a lot clearer if we renamed all the Materialize crates to have an `mz_` prefix. I've backed this out for now since it sounds like persist has its own convention. We'll fight the fight repo-wide one day when `rustfmt` can enforce this automatically. At the moment it's too hard to get everyone on board.